opentech/public/news/views.py

```diff
@@ -0,0 +1,46 @@
+from django.contrib.syndication.views import Feed
+
+from opentech.public.news.models import NewsPage, NewsType
+
+
+class NewsFeed(Feed):
+    link = "https://www.opentech.fund/news/"
+    title = "OTF – News, updates, and announcements"
+    description = "News, updates, and announcements from The Open Technology Fund"
```

Comment:
Will make these two in to settings.

Reply:
We can create the Feed Settings in the wagtail admin using [Site settings](https://docs.wagtail.io/en/v2.5/reference/contrib/settings.html) feature.

Reply:
Need feature and easy to implement.